content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -32,7 +32,7 @@ a certain policy and written to a backend. The policy determines what's recorded
 and the backends persist the records. The current backend implementations
 include logs files and webhooks.
 
-Each request can be recorded with an associated "stage". The known stages are:
```

Comment:
Can you fix the overview? A suggestion:

Kubernetes _Auditing_ provides a security-relevant, chronological set of records documenting
the sequence of activities [or actions] in a cluster. The cluster audits the activities generated by individual users, administrators or other components of the system. _Auditing_ allows cluster administrators to
answer the following questions:

Reply:
Good idea